src/app/modules/form/admin-form/admin-form.errors.ts

```diff
@@ -17,3 +17,9 @@ export class EditFieldError extends ApplicationError {
     super(message)
   }
 }
+
+export class FieldNotFoundError extends ApplicationError {
```

Comment:
i think the callsites for `FieldNotFoundError` have a variable called `fieldId` in scope and if we are to log this, should we also attach the variable `fieldId` to the constructor to aid in troubleshooting?

Reply:
no need; knowing the fieldId won't help since the field is not found in the database :P